src/java/org/apache/cassandra/db/commitlog/CommitLogReader.java

```diff
@@ -253,7 +253,7 @@ public void readCommitLogSegment(CommitLogReadHandler handler,
                     throw (IOException) re.getCause();
                 throw re;
             }
-            logger.debug("Finished reading {}", file);
+            logger.info("Finished reading {}", file);
```

Comment:
For which particular case was the logging for a finished commit log read bumped to INFO?
I can't think of cases when this would have helped me troubleshoot a cluster.

Reply:
After a non-graceful restart.  There are cases where the segment can get skipped and data missed, this can help identify data loss.

Reply:
kk